### ISSUE TITLE
In a Sequence Diagram  the primaryColor config is ignored for participants' background color 

### ISSUE THREAD

**Issue by ebouchut** [squidfunk/mkdocs-material#5034]
### Context

I configured a `primaryColor` in a sequence diagram on a page of a site using `mkdocs-material` version 9.0.12.
I expect the background color of the participant box to be the same as `primaryColor`, whereas it is not.

```
%%{
  init: {
    'theme': 'base',
      'themeVariables': {
        'primaryColor': 'lightgreen'
    }
  }
}%%
sequenceDiagram
    actor       user as User
    participant p    as Button

    user   ->>  p: click
```

### Bug description

The diagram is not rendered as expected:
- ✅ The background color of the **actor** box (`User`) is the same as `primaryColor` which is expected.
- ❌ The background color of the **participant** box (`Button`) is white, which is not expected, I 🤔 . 

Both Mermaid Live Editor and Github Previewer, use the `primaryColor` for the participant's background color. This is what I expected `mkdocs-material` to do.

My understanding of mermaid.js documentation is the following:
- [`actorBkg`](https://mermaid.js.org/config/theming.html#sequence-diagram-variables) mentions that `primaryColor` is the default background color for an actor.
- [`MainBkg`](https://mermaid.js.org/config/theming.html#theme-variables) computed from `primaryColor` "should be" used in this case. 

🤔 This is my understanding of this gray part of the documentation. 
Let me know what you think about this.

```mermaid
%%{
  init: {
    'theme': 'base',
      'themeVariables': {
        'primaryColor': 'lightgreen'
    }
  }
}%%
sequenceDiagram
    actor       user as User
    participant p    as Button

    user   ->>  p: click
```

### Related links

- [Same diagram in Mermaid Live](https://mermaid.live/edit#pako:eNpVUU1PwzAM_SuVpaqXMvV7bQ47AFdOCA4oF6_1tog2KWkiMar-d7xWIPAhefJ7z7HjGVrTEQgIw1lq6ZRWTgQrlC5yFxooEkF0xImi-G_2Fa3CY09T9CtnarRqQHt9ML2xN1-vzhd3tkQ62jTL7eJjCUOpJ_rwpFt6VHi2OEgdcGDrjA228BNtEKfghfGmGNE61aoRtQvGjbz3zhnNtfU_293hwHIRtL1q36WGGAayA6qOB55vWgnrMBIEw45O6HsngdtjKXpnnq-6BeGspxj82KH76RXECfuJs9wFiBk-QWTFflfXVZYWdVrmZVZnMVxBpHWyy5ukrphKqiapiiWGL2O4RLIrk32ZFWmT5XmesXOt97aS26PUKf6Op21H66qWb8K2ias)

### Reproduction

[example.zip](https://github.com/squidfunk/mkdocs-material/files/10716688/example.zip)

### Steps to reproduce

1. Open http://127.0.0.1:8000/diagram/
3. In the sequence diagram, the background color of **both** the **actor** (`User`) and the **participant** (`Button`) boxes should be green.

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting. Material for MkDocs Mermaid integration is a bit different – it automatically sets the colors to the [accent color](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#accent-color) you choose when configuring the theme. We use [CSS variables](https://github.com/squidfunk/mkdocs-material/blob/master/src/assets/stylesheets/main/integrations/_mermaid.scss) to override specific parts of Mermaid graphs. 

Fixed in 5065bf9cd. The sequence actors, which were rendered green in your example, are now also rendered using the defined accent color. This is consistent with the rendering of the rest of our Mermaid integration:

<img width="370" alt="Bildschirmfoto 2023-02-18 um 10 40 32" src="https://user-images.githubusercontent.com/932156/219853397-33736fbf-b655-42b5-a6aa-fa26f5aed271.png">

Additionally, this commit introduces a way to override accent colors on a per-diagram basis. When [`md_in_html`](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#markdown-in-html) is enabled, You can now wrap a Mermaid diagram with an accent color directive ([possible values defined here](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#accent-color)):

```` html
<div data-md-color-accent="green" markdown>

``` mermaid
sequenceDiagram
    actor       user     as User
    participant p   as Button

    user     ->>  p: click
```

</div>
````

If you wish to override colors for all Mermaid diagrams, use [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css) and override the variables defined here:

https://github.com/squidfunk/mkdocs-material/blob/5065bf9cd3b55f14a1770b99433385b8fe8cb155/src/assets/stylesheets/main/integrations/_mermaid.scss#L27-L41

**Comment by squidfunk** [comment]
Released as part of [9.0.13](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.13).

**Comment by ebouchut** [comment]
👍 Thank you!
